src/Table.luna

```diff
@@ -71,27 +85,128 @@ class Table:
     def schema:
         wrapper = self.ptr.schema
         SchemaVal wrapper
+
+    # Returns a single `Column` value from `Table` by integer index.
+    # 
+    # > import Dataframes.Column 
+    # > import Dataframes.Types
+    # > import Dataframes.Table 
+    # > l1 = [1,2,3,4,5]
+    # > l2 = [11,12,13,14,15]
+    # > col1 = Column.fromList "col1" Int64Type l1
+    # > col2 = Column.fromList "col2" Int64Type l2
+    # > table = Table.fromColumns [col1 , col2]
+    # > column = table.columnAt 1
+    #
+    # `index`: Integer column index.
+    # `return`: `Column` value from the `Table` for given index.
+
     def columnAt index:
         columnWrapper = self.ptr.columnAt index
         Column.fromColumnWrapper columnWrapper
+    
+    # Insert a single `Column` value to `Table` under given index.
+    # 
+    # > import Dataframes.Column 
+    # > import Dataframes.Types
+    # > import Dataframes.Table 
+    # > l1 = [1,2,3,4,5]
+    # > l2 = [11,12,13,14,15]
+    # > col1 = Column.fromList "col1" Int64Type l1
+    # > col2 = Column.fromList "col2" Int64Type l2
+    # > table = Table.fromColumns [col1 , col2]
+    # > l3 = [21,22,23,24,25]
+    # > col3 = Column.fromList "col3" Int64Type l3 
+    # > newTable = table.addColumn 1 col3
+    #
+    # `index`: Integer index for newly inserted column.
```

Comment:
Columns indexed from 0 or 1? What happens to the column that was on this position? Can you add a column on out of bounds index?